Minio.Functional.Tests/FunctionalTest.cs

```diff
@@ -1945,7 +1947,7 @@ private async static Task ListObjects_Test2(MinioClient minio)
             {
                 await Setup_Test(minio, bucketName);
 
-                ListObjects_Test(minio, bucketName, null, 0).Wait(1000);
+                ListObjects_Test(minio, bucketName, "", 0).Wait(1000);
```

Comment:
Do we have a test with `prefix` equal to `null` too?

Reply:
Added..